signl4/README.md

```diff
@@ -17,27 +17,74 @@ Follow these steps in SIGNL4:
+2. In your SIGNL4 app find your SIGNL4 webhook address including your team secret under *Teams -> Your Team -> Secret*.
 
-### Datadog
+### Datadog alerts
 
-Follow these steps in Datadog:
+You can alert your SIGNL4 team about new alerts in Datadog. Alerts that get resolved in Datadog automatically close the alert in SIGNL4. In order to do so you need to configure the following:
 
-1. Navigate to the [Webhooks Integration tile][6].
+1. Navigate to the [Webhooks Integration tile][9].
 
+2. On the **Configuration** tab, go to Webhooks and click **New**.
 
+3. Under **New Webhook**, enter a meaningful `Name` and use the SIGNL4 Webhook `URL` (created above) including your team secret, for example:
+
+    ```
+    https://connect.signl4.com/webhook/[team-secret]?ExtIDParam=alertId&ExtStatusParam=alertTransition&ResolvedStatus=Recovered
+    ```
 
-2. On the **Configuration** tab, scroll down and click **New**.
+    Replace `[team-secret]` with your SIGNL4 team secret here.
+
+    ![SIGNL4 Alerts Webhook][3]
+
+4. Copy-paste the following JSON in the `Payload` text box:
+
+    ```json
+	{
+		"title": "$EVENT_TITLE",
+		"message": "$TEXT_ONLY_MSG",
+		"link": "$LINK",
+		"priority": "$ALERT_PRIORITY",
+		"host": "$HOSTNAME",
+		"alertScope": "$ALERT_SCOPE",
+		"alertStatus": "$ALERT_STATUS",
+		"alertId": "$ALERT_ID",
+		"alertTransition": "$ALERT_TRANSITION",
+		"X-S4-SourceSystem": "Datadog",
+		"date": "$DATE",
+		"org": {
+			"id": "$ORG_ID",
+			"name": "$ORG_NAME"
+		},
+		"id": "$ID"
+	}
+    ```
+
+You can adapt the parameters according to your needs, but leave `alertId`, `alertTransition`, and `X-S4-SourceSystem` unchanged.
+
+5. Click **Save** to create the webhook.
+
+For more details, see [Mobile alerting with tracking & escalation for Datadog][7].
+
+You can now use your webhook as a notification channel in your monitor. Assuming the name of your webhook is SIGNL4, send notifications using `@webhook-SIGNL4`. When the conditions of the monitor apply, your team receives a new SIGNL4 alert.
+
+### Datadog Incidents
+
+You can alert your SIGNL4 team about new Incidents in Datadog. Incidents that get resolved in Datadog automatically close the alert in SIGNL4. In order to do so, configure the following:
+
+1. Navigate to the [Webhooks Integration tile][9].
+
+2. On the **Configuration** tab, go to Webhooks and click **New**.
 
 3. Under **New Webhook**, enter a meaningful `Name` and use the SIGNL4 Webhook `URL` (created above) including your team secret, for example:
 
     ```
-    https://connect.signl4.com/webhook/<team-secret>?ExtIDParam=alertId&ExtStatusParam=alertTransition&ResolvedStatus=Recovered
+    https://connect.signl4.com/webhook/[team-secret]?ExtIDParam=s4ExternalId&ExtStatusParam=incidentStatus&ResolvedStatus=resolved
     ```
 
-    Replace `<team-secret>` with your SIGNL4 team secret here.
+    Replace `[team-secret]` with your SIGNL4 team secret here.
```

Comment:
```suggestion
    Replace `[team-secret]` in the URL with your SIGNL4 team secret.
```

Reply:
Thanks @yzhan289. Suggestion approved and I hope all is fine now.